### PR TITLE
WinUI-2902 : Added Content in BusyContent BusyIndicator control.

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description:
Prepared BusyContent and customization MD file in BusyIndicator control.

### CODE REVIEW

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/BusyIndicator/Content.md [hunk 1]
```diff
@@ -1,12 +1,80 @@
 ---
 layout: post
-title: Content with WinUI BusyIndicator control | Syncfusion
-description: Learn here all about showing the loading progress of the content with Syncfusion WinUI BusyIndicator Control.
+title: Content in WinUI BusyIndicator control | Syncfusion
+description: Learn here all about the loading progress content and its customization available in the Syncfusion WinUI BusyIndicator control.
```
Comment (OnyimbiNelson) [inline winui/BusyIndicator/Content.md:4]: description: Learn all about the loading progress content and its customization available in the Syncfusion WinUI BusyIndicator control here.

File: winui/BusyIndicator/Content.md [hunk 2]
```diff
@@ -1,12 +1,80 @@
...
 platform: WinUI
 control: SfBusyIndicator
 documentation: ug
 ---
 
 # Content in WinUI BusyIndicator
 
-This section explains the content features available in the WinUI BusyIndicator control.
+The BusyIndicator control provides option to set the custom message that indicates the busy status of the control to the users by using the `BusyContent` property.
```
Comment (OnyimbiNelson) [inline winui/BusyIndicator/Content.md:12]: The BusyIndicator control provides an option to set the custom message that indicates the busy status of the control to the users by using the `BusyContent` property.

File: winui/BusyIndicator/Content.md [hunk 3]
```diff
@@ -1,12 +1,80 @@
...
+busyIndicator.BusyContentPosition = BusyIndicatorContentPosition.Top;
+
+{% endhighlight %}
+{% endtabs %}
+
+![BusyContentPosition in WinUI BusyIndicator control.](BusyIndicator_videos/winui_busyindicator_contentposition.jpg)
+
+## Busy content template
+
+The BusyIndicator control provides options to customize the content by using the `DataTemplate` with the help of `BusyContentTemplate` property.
```
Comment (OnyimbiNelson) [inline winui/BusyIndicator/Content.md:64]: The BusyIndicator control provides options to customize the content by using the `DataTemplate` with the help of the `BusyContentTemplate` property.

File: winui/BusyIndicator/Customization.md [hunk 4]
```diff
@@ -1,12 +1,87 @@
 ---
 layout: post
-title: Customization with WinUI BusyIndicator control | Syncfusion
-description: Learn here all about UI customization with Syncfusion WinUI BusyIndicator Control (SfBusyIndicator), its elements, and more.
+title: Customization in WinUI BusyIndicator control | Syncfusion
+description: Learn here all about the customization features such as size factor, duration factor, and color in Syncfusion WinUI BusyIndicator (SfBusyIndicator) control.
```
Comment (OnyimbiNelson) [inline winui/BusyIndicator/Customization.md:4]: description: Learn all about the customization features such as size factor, duration factor, and color in Syncfusion WinUI BusyIndicator (SfBusyIndicator) control here.

File: winui/BusyIndicator/Content.md (hunk 1 above)
Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/Content.md:4]: Changed the description content as mentioned above.

File: winui/BusyIndicator/Content.md (hunk 2 above)
Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/Content.md:12]: Changed the content as mentioned above.

File: winui/BusyIndicator/Customization.md (hunk 4 above)
Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/Customization.md:4]: Changed the customization content as mentioned above.

**Pull request merged**